cot-macros/tests/compile_tests.rs

```diff
@@ -50,3 +50,11 @@ fn attr_main() {
     t.pass("tests/ui/attr_main.rs");
     t.compile_fail("tests/ui/attr_main_args.rs");
 }
+
+#[rustversion::attr(not(nightly), ignore)]
+#[test]
+#[cfg_attr(miri, ignore)] // unsupported operation: extern static `pidfd_spawnp` is not supported by Miri
+fn derive_from_struct() {
+    let t = trybuild::TestCases::new();
+    t.pass("tests/ui/derive_from_request_parts.rs");
```

Comment:
Note that this causes the file only to compile; the tests inside `derive_from_request_parts.rs` are still not run, unless there is a `main` function. If you want the tests to be run, please move the file to `cot-macros/tests`, as mentioned in [my other comment](https://github.com/cot-rs/cot/pull/336/files#r2109045844).